WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -323,17 +323,14 @@ The response is always zero (0) without authorization.
   [locationManager setPausesLocationUpdatesAutomatically:NO];
   [locationManager startUpdatingLocation];
 
-  CLAuthorizationStatus authStatus;
-  if ([locationManager respondsToSelector:@selector(authorizationStatus)]) {
-    authStatus = locationManager.authorizationStatus;
-  } else {
-    authStatus = [CLLocationManager authorizationStatus];
-  }
-  CLLocation *location = locationManager.location;
+  CLAuthorizationStatus authStatus = [locationManager respondsToSelector:@selector(authorizationStatus)]
+  ? locationManager.authorizationStatus
+  : [CLLocationManager authorizationStatus];
+
   return FBResponseWithObject(@{
     @"authorizationStatus": @(authStatus),
-    @"latitude": @(location.coordinate.latitude),
-    @"longitude": @(location.coordinate.longitude),
+    @"latitude": @(locationManager.location.coordinate.latitude),
```

Comment:
is it possible that latitude/longiture value is going to be nil if the manager is not authorized?

Reply:
No. Then, they are `0`. When the location service is off case as well